src/public/interfaces.ts

```diff
@@ -307,6 +307,11 @@ export interface Context {
    * Current ring ID
    */
   ringId?: string;
+
+  /**
+   * Unique ID for the current Tab session for use in correlating telemetry data.
+   */
+  tabSessionId?: string;
```

Comment:
Could we call this `appSessionId`? We generally don't explicitly refer to tabs in the SDK since it supports many more scenarios than just tabs (e.g. task modules, connectors, etc.).

Reply:
Changed it Yuri